src/Storage/Storage.Management/StorageAccount/UpdateAzureStorageAccountNetworkRuleSet.cs

```diff
@@ -126,6 +126,8 @@ public override void ExecuteCmdlet()
         {
             base.ExecuteCmdlet();
 
+            WriteWarning("The DefaultAction value will change in a future release from: Allow = 1, Deny = 0, to: Allow = 0, Deny = 1.");
```

Comment:
Please refer to the link for your breaking change case https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#generic-change-in-a-parameter.

Reply:
Fixed